rules/filename-case.js

```diff
@@ -9,6 +9,29 @@ var pascalCase = function (str) {
 	return upperfirst(camelCase(str));
 };
 
+var numberRegex = /(\d+)/;
+var PLACEHOLDER = 'œœœœœœœœœœ';
```

Comment:
This is a bit of a hack, but could not find any (open to suggestions) placeholders that would remain after being snakeCased or anything.